sdk/js/index.js

```diff
@@ -44,7 +70,7 @@ const runGame = (agent, gameAddress, formatAddress) => {
   // very large?
   // - no, game contributors have the ability to factor address game
   // data however they want but they don't have that ability with formats
-  return runner(agent, formatAddress, result.Ok.Game.data)
+  return runner(agentId, formatAddress, JSON.parse(result.Ok.Game.data))
```

Comment:
I think I'd rather let the runner be responsible for parsing the game data. This would move the parsing to the beginning of the specific runner.